java-logging-appinsights/src/main/resources/logback-includes.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<included>
+    <appender name="APP_INSIGHTS" class="com.microsoft.applicationinsights.logback.ApplicationInsightsAppender">
+        <instrumentationKey>${APPLICATION_INSIGHTS_IKEY:-00000000-0000-0000-0000-000000000000}</instrumentationKey>
```

Comment:
does it need a default?

Reply:
probably not. application will fail to start eitherway. if not here, than elsewhere. i'll remove default